lib/matplotlib/tests/test_colorbar.py

```diff
@@ -298,6 +298,26 @@ def test_colorbar_ticks():
     assert len(cbar.ax.xaxis.get_ticklocs()) == len(clevs)
 
 
+@cleanup
+def test_colorbar_get_ticks():
+    # test feature for #5792
+    plt.figure()
+    data = np.arange(1200).reshape(30, 40)
+
+    plt.subplot()
+    plt.contourf(data)
```

Comment:
For the test, it would be better to use the more OO style instead of the pyplot state machine style.